src/main/kotlin/com/personio/synthetics/client/BrowserTest.kt

```diff
@@ -12,10 +12,14 @@ import com.personio.synthetics.config.loadConfiguration
 
 /**
  * Creates a synthetic browser test with the added configurations (if any) and steps in Datadog
- * The default configurations for the tests are loaded from the configuration file in resources
+ * The default configurations for the tests are loaded from the configuration file in resources.
+ *
+ * Deprecation notice: In future releases this method will be replaced by one that
+ * uses SyntheticTestBuilder or similar.
  * @param name Name of the test
  * @param steps Calls the added steps and configuration functions of the test
  */
+@Deprecated(message = "Planned for removal in 3.x.x")
```

Comment:
can we update the depreciation message with the replace method and deprecation level as well?

```
@Deprecated(
    message = "Planned for removal in 3.x.x",
    replaceWith = ReplaceWith(
        expression = "Function/class",
        imports = ["corresponding class"]
    ),
    level = DeprecationLevel.WARNING
)
```

Reply:
There is no replacement right now, which is why I haven't included this info.